modules/touch/src/win32_touch.cpp

```diff
@@ -213,8 +218,10 @@ Win32TouchHook::Win32TouchHook(void* nativeWindow)
 
     if (!gStarted) {
         gStarted = true;
-        gStartTime = std::chrono::duration_cast<std::chrono::microseconds>(
-            std::chrono::high_resolution_clock::now().time_since_epoch()
+        LARGE_INTEGER count;
```

Comment:
This seems like a smart thing, but what makes it better than the previous solution with high-resolution-clock.time-since-epoch? Is it something we want to incorporate in more places where we currently use high-res-clock?

Reply:
Good question!

std::chrono might use the built-in windows call under-the-hood, but for this specific windows touch hook we get the value for each touch from the touch display driver as a windows-supplied time, meaning that if we want to measure against the time received from the touch input I felt like it would be best to get the "startTime" from the windows API as well, so they match.

I don't think this should spread to any other places, it might be relevant if we want to provide our own timer classes and not rely on std::chrono, but I feel like that's an issue that needs another problem than this one.

Reply:
Looking at https://github.com/microsoft/STL/blob/master/stl/inc/chrono it seems like microsoft STL uses basically the exact same code, so it shouldn't matter, want me to change it back?

Reply:
Then I would suggest to change it back to keep it more consistent with other parts of the app that uses high_resolution_clock.